Makefile

```diff
@@ -66,18 +66,18 @@ check-origin-remote:
 	# A remote named origin MUST exist.
 	@git remote show origin
 
+dist: clean ## build source and wheel packages
```

Comment:
Oh! I'm really sorry, I didn't mean to say that the `dist` target should come before the `tag-release` target in general, just in the target list for `release` like
```make
release: dist tag-release
    # ...
```
That way I think I'd avoid accidentally tagging a release if an error occurred in building a dist.

If you wouldn't mind moving the `dist` target back to where it was originally then I'll really be happy to merge this. Promise! 😅

Reply:
If you could also squash these and any subsequent commits that would be great. 😁